src/Microsoft.PowerShell.Commands.Utility/resources/WebCmdletStrings.resx

```diff
@@ -133,7 +133,10 @@
     <value>The provided JSON includes a property whose name is an empty string, this is only supported using the -AsHashTable switch.</value>
   </data>
   <data name="DuplicateKeysInJsonString" xml:space="preserve">
-    <value>Cannot convert the JSON string because a dictionary that was converted from the string contains the duplicated keys '{0}' and '{1}'.</value>
+    <value>Cannot convert the JSON string because a dictionary that was converted from the string contains the duplicated key '{0}'.</value>
+  </data>
+  <data name="KeysWithDifferentCasingInJsonString" xml:space="preserve">
+    <value>Cannot convert the JSON string because a PSObject does not support keys with different casing. Please use the -AsHashTable switch instead. The key that was attempted to be added to the exisiting key '{0}' was '{1}'.</value>
```

Comment:
Oh, very long message!

Reply:
I know. I shortened it a bit but I think the last sentence still adds value.